### PR TITLE
Search bar debug

### PR DESCRIPTION
**Pull request by jsong336**

### CODE REVIEW

File: _client/livethere-app/src/components/search/searchBar.js [hunk 1]
```diff
@@ -94,15 +146,16 @@ const Search = (props) => {
             placeholder="Search your university/college"
             onChange={handleInputChange}
             onKeyUp={handleKeyPress}
+            onClick={()=>{ if (keyword !== '') setShowDropbox(true)}}
```
Comment (hPark0811) [inline _client/livethere-app/src/components/search/searchBar.js:149]: can you just make sure codes are in similar format? You can get jsx formatted or react formatted to do them. We have different spaces here and there.

Comment (hPark0811) [reply _client/livethere-app/src/components/search/searchBar.js:149]: ()=>{ if } to be () => { if }

File: _client/livethere-app/src/components/search/searchBar.js [hunk 2]
```diff
@@ -56,28 +62,74 @@ const Search = (props) => {
...
+        const [key, val] = matchingTuples[dropDownIndx];
+        handleSearchByKey(key);
+      }
+      else{
+        // If dropdown index is not selected or drop box is not shown, search by keyword
+        handleSearchByKeyword(keyword);
+      }
+    }
+    if (event.key ==='ArrowUp'){
+      selectDropdown(dropDownIndx-1);
```
Comment (hPark0811) [inline _client/livethere-app/src/components/search/searchBar.js:111]: dropDownIndx - 1 pls and thank you

File: _client/livethere-app/src/components/search/searchBar.js (hunk 1 above)
Comment (jsong336) [reply _client/livethere-app/src/components/search/searchBar.js:149]: just "believe"

**Pull request merged**